### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by aconchillo**
#### Please describe the changes in your PR. If it is addressing an issue, please reference that as well.

The main update is `numpy` to version 2 which is supposed to be faster and `audioop-lts` which is a copy of the python < 3.12 `audioop` module which is now deprecated in python 3.13.

### CODE REVIEW

**Comment by markbackman** [comment]
From a clean venv, I get this conflict when running `pip install -r dev-requirements.txt`:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pyht 0.1.4 requires protobuf<5.0.0,>=4.24.3, but you have protobuf 5.29.1 which is incompatible.
pipecat-ai 0.0.50.dev162 requires protobuf~=4.25.4, but you have protobuf 5.29.1 which is incompatible.
```

I tried downgrading grpcio-tools to 1.65.4 and I no longer get the conflict.

**Comment by markbackman** [comment]
Ok, easy fix. Just bump the playht version to:
```
playht = [ "pyht~=0.1.9", "websockets~=13.1" ]
```

**Comment by aconchillo** [comment]
> Ok, easy fix. Just bump the playht version to:
> 
> ```
> playht = [ "pyht~=0.1.9", "websockets~=13.1" ]
> ```

Done!

**Review by markbackman — APPROVED** [review]
LGTM!

File: pyproject.toml [hunk 1]
```diff
@@ -21,12 +21,14 @@ classifiers = [
...
 dependencies = [
     "aiohttp~=3.11.10",
-    "loguru~=0.7.2",
+    "audioop-lts~=0.2.1; python_version>='3.13'",
+    "loguru~=0.7.3",
     "Markdown~=3.7",
-    "numpy~=1.26.4",
-    "Pillow~=10.4.0",
+    "numpy~=2.1.3",
+    "numba~=0.61.0rc1",
```
Comment (aconchillo) [inline pyproject.toml:28]: Btw, this is necessary because `resampy` depends on `numba` but we need a `numba` that's compatible with `numpy` 2.

**Pull request merged**